lib/components/molecules/StickyHeader/StickyHeader.css

```diff
@@ -0,0 +1,8 @@
+.sticky .sticky-inner {
+  position: fixed;
+  top: 0;
+  left: 0;
+  right: 0;
+  z-index: 1;
```

Comment:
z-index should be picked from Design system tokens

Reply:
Fixed.